### PR TITLE
docs(commands): update docs to show NPM commands where there are Yarn commands

### PR DESCRIPTION
**Pull request by davemneo**
Updating docs to include npm commands for every yarn command.

### Requirements

provide appropriate npm commands for every yarn command. 

### Test Plan
manually tested:
```    npm install @neo-one/client @neo-one/cli @neo-one/smart-contract @neo-one/smart-contract-test @neo-one/smart-contract-typescript-plugin
npm run neo-one init
npm install
npm run neo-one build
npm run start
npm test
```

### Applicable Issues

re #781

### CODE REVIEW

File: packages/neo-one-website/tutorial/tutorial.md [hunk 1]
```diff
@@ -101,11 +104,11 @@ This smart contract doesn't do a whole lot, in fact it does nothing, but this is
   4. Deploy the smart contracts to the private NEO network.
   5. Generate code for use in your decentralized app.
 
-Additionally, you may run `yarn build` with the `--reset` flag to reset the network. Running with `--watch` will execute the above process whenever you make a change to your smart contracts.
+Additionally, you may run `yarn build` OR `npm build` with the `--reset` flag to reset the network. Running with `--watch` will execute the above process whenever you make a change to your smart contracts.
```
Comment (dicarlo2) [inline packages/neo-one-website/tutorial/tutorial.md]: This should actually be `yarn neo-one build`

Comment (davemneo) [reply packages/neo-one-website/tutorial/tutorial.md]: 👍

File: packages/neo-one-website/docs/0-installation/2-playground.md [hunk 2]
```diff
@@ -43,6 +45,7 @@ This will start up a local network, compile the smart contracts located in the `
 
 ```bash
 yarn start
+# npm run ts-node ./scripts/start.ts
```
Comment (dicarlo2) [inline packages/neo-one-website/docs/0-installation/2-playground.md]: `npm run start` doesn't work here? If not, then it should probably be `npm run-script start`

File: packages/neo-one-website/docs/0-installation/2-playground.md [hunk 3]
```diff
@@ -25,6 +25,7 @@ This guide will walk you through getting started with the [NEO•ONE Playground]
 git clone https://github.com/neo-one-suite/neo-one-playground.git
 cd neo-one-playground
 yarn install
+# npm install
```
Comment (dicarlo2) [inline packages/neo-one-website/docs/0-installation/2-playground.md]: I think it'd be better to put each of these above the `yarn` command - generally comments precede the article they're referring to.

File: packages/neo-one-website/docs/0-installation/2-playground.md (hunk 2 above)
Comment (davemneo) [reply packages/neo-one-website/docs/0-installation/2-playground.md]: I'm pretty sure I tried that and it failed with some errors.  I just verified that "npm run start" works.

File: packages/neo-one-website/docs/0-installation/2-playground.md (hunk 3 above)
Comment (davemneo) [reply packages/neo-one-website/docs/0-installation/2-playground.md]: 👍

File: packages/neo-one-website/docs/0-installation/2-playground.md (hunk 2 above)
Comment (davemneo) [reply packages/neo-one-website/docs/0-installation/2-playground.md]: 👍

**Pull request merged**